### PR TITLE
move infection and structurizr in docker

### PR DESCRIPTION
**Pull request by Derane**
## Description
move infection and structurizr in docker

## Related Issue

https://github.com/VilnaCRM-Org/php-service-template/issues/26

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have performed a self-review of my code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING.md**](https://github.com/VilnaCRM-Org/php-service-template/blob/main/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] You have only one commit (if not, squash them into one commit).

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -95,6 +96,7 @@
         "doctrine/doctrine-fixtures-bundle": "^3.4",
         "friends-of-behat/symfony-extension": "^2.4",
         "friendsofphp/php-cs-fixer": "^3.14",
+        "infection/infection": "^0.27.10",
```
Comment (Kravalg) [inline composer.json:99]: It will be better to write the latest version 0.29

File: .env [hunk 2]
```diff
@@ -12,3 +12,5 @@ APP_SECRET=2e222659c5b006df8d0e68cc33913706
 # DATABASE_URL="postgresql://app:!ChangeMe!@127.0.0.1:5432/app?serverVersion=15&charset=utf8"
 DATABASE_URL=
 ###< doctrine/doctrine-bundle ###
+
+STRUCTURIZR_PORT=
```
Comment (Kravalg) [inline .env:16]: It will be better to write the default port there

File: infection.json5 [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "vendor/infection/infection/resources/schema.json",
+  "source": {
+    "directories": [
+      "src"
+    ],
```
Comment (Kravalg) [inline infection.json5:6]: It will be better to remove this, because we removed the DataFixtures from the src directory

**Review by Kravalg — CHANGES_REQUESTED** [review]
Good job, I left several comments

**Review by Kravalg — APPROVED** [review]
Looks good, please fix the conflicts and then we will be able to merge the PR

**Review by Kravalg — CHANGES_REQUESTED** [review]
Let's fix the CI and we will be able to merge this PR

**Comment by Derane** [comment]
> Let's fix the CI and we will be able to merge this PR

I configured mutations in 'move shared' pull request, because here we have no tests

**Review by Kravalg — APPROVED** [review]

**Pull request merged**